zzzz_modular_occulus/code/modules/reagents/reagents/food-Drinks.dm

```diff
@@ -0,0 +1,359 @@
+	description = "A deep chocolate flavour with a smooth blend of cream and milk, shaked and cooled to perfection in order to guarantee palate satisfaction, all topped with a light layer of froth."
+	taste_description = "rich creamy chocolate"
+	id = "chocolate_milkshake"
+	color = "#81551D"
+	adj_temp = -9
+	taste_tag = list(TASTE_LIGHT, TASTE_SWEET, TASTE_SLIMEY)
+
+	glass_icon_state = "shake_chocolate"
+	glass_name = "Chocolate Milkshake"
+	glass_desc = "A deep chocolate flavour with a smooth blend of cream and milk, shaked and cooled to perfection in order to guarantee palate satisfaction, all topped with a light layer of froth."
+	glass_center_of_mass = list("x"=16, "y"=7)
+
+/datum/reagent/drink/milkshake/chocolate/New()
+	var/toppings = pick("_solo", "_straw", "_umbr", "_both")
+	glass_icon_state += toppings
+
+/datum/reagent/drink/milkshake/mintchocolate // We have no mint plants.
+	name = "Mint Chocolate Milkshake"
+	description = "A refreshing mint chocolate flavour with a smooth blend of cream and milk, shaked and cooled to perfection in order to guarantee palate satisfaction, all topped with a light layer of froth."
+	taste_description = "cooling creamy mint chocolate"
+	id = "mintchocolate_milkshake"
+	color = "#9F7136"
+	adj_temp = -9
+	taste_tag = list(TASTE_LIGHT, TASTE_SWEET, TASTE_REFRESHING, TASTE_SLIMEY)
+
+	glass_icon_state = "shake_chocolate_mint"
+	glass_name = "Mint Chocolate Milkshake"
+	glass_desc = "A refreshing mint chocolate flavour with a smooth blend of cream and milk, shaked and cooled to perfection in order to guarantee palate satisfaction, all topped with a light layer of froth."
+	glass_center_of_mass = list("x"=16, "y"=7)
+
+/datum/reagent/drink/milkshake/mintchocolate/New()
+	var/toppings = pick("_solo", "_straw", "_umbr", "_both")
+	glass_icon_state += toppings
+
+/datum/reagent/drink/milkshake/banana
+	name = "Banana Milkshake"
+	description = "A sweet and gentle banana flavour with a smooth blend of cream and milk, shaked and cooled to perfection in order to guarantee palate satisfaction, all topped with a light layer of froth."
+	taste_description = "smooth creamy banana"
+	id = "banana_milkshake"
+	color = "#FFFD78"
+	adj_temp = -9
+	taste_tag = list(TASTE_LIGHT, TASTE_SWEET, TASTE_SLIMEY)
+
+	glass_icon_state = "shake_banana"
+	glass_name = "Banana Milkshake"
+	glass_desc = "A sweet and gentle banana flavour with a smooth blend of cream and milk, shaked and cooled to perfection in order to guarantee palate satisfaction, all topped with a light layer of froth."
+	glass_center_of_mass = list("x"=16, "y"=7)
+
+/datum/reagent/drink/milkshake/banana/New()
+	var/toppings = pick("_solo", "_straw", "_umbr", "_both")
+	glass_icon_state += toppings
+
+/datum/reagent/drink/milkshake/berry
+	name = "Berry Milkshake"
+	description = "A sweet forrest berry flavour with a smooth blend of cream and milk, shaked and cooled to perfection in order to guarantee palate satisfaction, all topped with a light layer of froth."
```

Comment:
Forrest with two Rs is a given name, forest refers to the large tracts of trees.

Reply:
A